### PR TITLE
Build Portal API Documentation: SshKey Module

### PR DESCRIPTION
**Pull request by chittaranjan-behera**
Build Portal API Documentation: SshKey Module

### CODE REVIEW

File: ssh-key/add.md [hunk 1]
```diff
@@ -41,15 +41,30 @@ NocWorx-Api-Version: 0.0.0
...
+  "input": {},
+  "request_date": 1638435978,
+  "resolved_date": null,
+  "resource": null,
   "staff_user": null,
-  "user": { "id": 27767, "identity": "Annabel Whyman - user@nocworx.com" },
-  "uuid": "6bf71868-d4ea-486a-95b9-0576ad3ef984"
+  "user": {
+    "id": 61420,
+    "identity": "Nexcess Staff - nocworx-dev@nexcess.net",
```
Comment (adrian-enspired) [inline ssh-key/add.md:61]: use a fake customer name (Annabel was fine).

File: ssh-key/delete.md [hunk 2]
```diff
@@ -37,15 +37,37 @@ NocWorx-Api-Version: 0.0.0
...
-  "request_date": 1566327634,
-  "resource_uri": "",
+  "api_version": "0.5.0",
+  "errors": {},
+  "input": {},
+  "request_date": 1638436250,
+  "resolved_date": null,
+  "resource": {
+    "id": 140,
+    "identity": "example-key - SHA256:3XTUdV3yYM/42K7a9ZP4V/XPUmeXO2/6PX1q8Ml3Mws chittaranjanbehera@it-HP-ProBook-440-G7 (RSA)",
```
Comment (adrian-enspired) [inline ssh-key/delete.md:55]: let's cut out the middle of that.
```
"identity": "example-key - SHA256:3XTUdV3yYM/. . . (RSA)",
```

File: ssh-key/delete.md [hunk 3]
```diff
@@ -37,15 +37,37 @@ NocWorx-Api-Version: 0.0.0
...
+      "scope": "user-ssh-key",
+      "uri": "/v1/ssh-key/140"
+    }
+  },
   "staff_user": null,
-  "user": { "id": 27767, "identity": "Annabel Whyman - user@nocworx.com" },
-  "uuid": "f2805dd2-077e-44f9-8302-3a371c7d532d"
+  "user": {
+    "id": 61420,
+    "identity": "Nexcess Staff - nocworx-dev@nexcess.net",
```
Comment (adrian-enspired) [inline ssh-key/delete.md:64]: use a fake customer name.

File: ssh-key/list.md [hunk 4]
```diff
@@ -31,16 +31,34 @@ curl -i "$PORTAL_API_URL/v1/ssh-key" \
 ```
 [
   {
-    "id": 138,
-    "identity": "stark.pub - f4:e1:75:e9:cf:83:c4:0b:83:e8:36:6b:c5:61:22:8f  user@example.com (RSA)",
-    "client": { "id": 24512, "identity": "Bosco-Bradtke" },
-    "creation_date": 1490204568,
-    "fingerprint": "f4:e1:75:e9:cf:83:c4:0b:83:e8:36:6b:c5:61:22:8f  user@example.com (RSA)",
+    "id": 139,
+    "identity": "nexcess-ssh-key - SHA256:3XTUdV3yYM/42K7a9ZP4V/XPUmeXO2/6PX1q8Ml3Mws chittaranjanbehera@it-HP-ProBook-440-G7 (RSA)",
```
Comment (adrian-enspired) [inline ssh-key/list.md:35]: `. . .` in the middle

File: ssh-key/list.md [hunk 5]
```diff
@@ -31,16 +31,34 @@ curl -i "$PORTAL_API_URL/v1/ssh-key" \
...
+      "id": 38114,
+      "identity": "Howell Inc",
+      "metadata": {
+        "scope": "client",
+        "uri": null
+      },
+      "status": "active"
+    },
+    "creation_date": 1638366250,
+    "fingerprint": "SHA256:3XTUdV3yYM/42K7a9ZP4V/XPUmeXO2/6PX1q8Ml3Mws chittaranjanbehera@it-HP-ProBook-440-G7 (RSA)",
```
Comment (adrian-enspired) [inline ssh-key/list.md:50]: `. . .` in the middle

File: ssh-key/list.md [hunk 6]
```diff
@@ -31,16 +31,34 @@ curl -i "$PORTAL_API_URL/v1/ssh-key" \
...
     "key_size": 2048,
-    "name": "stark.pub",
-    "user": { "id": 27767, "identity": "Annabel Whyman - user@example.com" }
+    "name": "nexcess-ssh-key",
+    "user": {
+      "id": 61420,
+      "identity": "Nexcess Staff - nocworx-dev@nexcess.net",
```
Comment (adrian-enspired) [inline ssh-key/list.md:55]: fake customer name

File: ssh-key/show.md [hunk 7]
```diff
@@ -35,15 +35,33 @@ NocWorx-Api-Version: 0.0.0
...
 
 {
-  "id": 138,
-  "identity": "stark.pub - f4:e1:75:e9:cf:83:c4:0a:83:e8:26:6b:c5:61:22:8f  user@example.com (RSA)",
-  "client": { "id": 24512, "identity": "Bosco-Bradtke" },
-  "creation_date": 1490204568,
-  "fingerprint": "f4:e1:75:e9:cf:83:c4:0a:83:e8:26:6b:c5:61:22:8f  user@example.com (RSA)",
-  "key": "ssh-rsa AAAAB3NzaC1yc2EAAAADCQABAAABAQDDixNxEbVT . . . user@example.com",
+  "id": 139,
+  "identity": "nexcess-ssh-key - SHA256:3XTUdV3yYM/42K7a9ZP4V/XPUmeXO2/6PX1q8Ml3Mws chittaranjanbehera@it-HP-ProBook-440-G7 (RSA)",
```
Comment (adrian-enspired) [inline ssh-key/show.md:39]: `. . .`

File: ssh-key/show.md [hunk 8]
```diff
@@ -35,15 +35,33 @@ NocWorx-Api-Version: 0.0.0
...
+    "id": 38114,
+    "identity": "Howell Inc",
+    "metadata": {
+      "scope": "client",
+      "uri": null
+    },
+    "status": "active"
+  },
+  "creation_date": 1638366250,
+  "fingerprint": "SHA256:3XTUdV3yYM/42K7a9ZP4V/XPUmeXO2/6PX1q8Ml3Mws chittaranjanbehera@it-HP-ProBook-440-G7 (RSA)",
```
Comment (adrian-enspired) [inline ssh-key/show.md:54]: `. . .`

File: ssh-key/show.md [hunk 9]
```diff
@@ -35,15 +35,33 @@ NocWorx-Api-Version: 0.0.0
...
   "key_size": 2048,
-  "name": "stark.pub",
-  "user": { "id": 27767, "identity": "Annabel Whyman - user@nocworx.com" }
+  "name": "nexcess-ssh-key",
+  "user": {
+    "id": 61420,
+    "identity": "Nexcess Staff - nocworx-dev@nexcess.net",
```
Comment (adrian-enspired) [inline ssh-key/show.md:59]: use fake customer name

**Review by adrian-enspired — CHANGES_REQUESTED** [review]

File: ssh-key/add.md (hunk 1 above)
Comment (chittaranjan-behera) [reply ssh-key/add.md]: Used fake customer name

File: ssh-key/delete.md (hunk 2 above)
Comment (chittaranjan-behera) [reply ssh-key/delete.md]: Removed the middle part.

File: ssh-key/delete.md (hunk 3 above)
Comment (chittaranjan-behera) [reply ssh-key/delete.md]: Used fake customer name

File: ssh-key/list.md (hunk 4 above)
Comment (chittaranjan-behera) [reply ssh-key/list.md]: Removed middle part

File: ssh-key/list.md (hunk 5 above)
Comment (chittaranjan-behera) [reply ssh-key/list.md]: Removed middle part

File: ssh-key/list.md (hunk 6 above)
Comment (chittaranjan-behera) [reply ssh-key/list.md]: Used fake customer name

File: ssh-key/show.md (hunk 7 above)
Comment (chittaranjan-behera) [reply ssh-key/show.md]: Removed middle part

File: ssh-key/show.md (hunk 8 above)
Comment (chittaranjan-behera) [reply ssh-key/show.md]: Removed middle part

File: ssh-key/show.md (hunk 9 above)
Comment (chittaranjan-behera) [reply ssh-key/show.md]: Used fake customer name

**Review by adrian-enspired — APPROVED** [review]

**Pull request merged**